api/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.jboss.weld</groupId>
 		<artifactId>weld-parent</artifactId>
-		<version>31</version>
+		<version>36</version>
```

Comment:
Weld itself (as a CDI impl) depends on `weld-parent`. Because `weld-parent` is basically just a declaration of plugin versions and nothing more. So I don't really see it as a problem.